README.md

```diff
@@ -16,7 +44,7 @@ An integrated suite of powertools for Lambda functions to make it effortless for
 
 * [logger](/packages/lambda-powertools-logger): structured logging with JSON, configurable log levels, and integrates with other tools to support correlation IDs and sampling (only enable debug logs on 1% of invocations)
 
-* [correlation IDs](/packages/lambda-powertools-correlation-ids): create and store correlation IDs that follow our naming convention
+* [correlation IDs](/packages/lambda-powertools-correlation-ids): create and store correlation IDs that follow the DAZN naming convention
```

Comment:
x-correlation-id is a convention outside of DAZN's world, might be worth changing to just
```suggestion
* [correlation IDs](/packages/lambda-powertools-correlation-ids): create and store correlation IDs that follow naming conventions
```

Reply:
Hrm — I'd argue we're either more explicit, or omit that line. Referencing a "naming convention" but not calling it out by name doesn't seem very helpful to me?